docs/training-guide.md

```diff
@@ -8,90 +8,101 @@ nav_order: 2
 
 A step-by-step guide on how to train a translation model.
 
-The configuration of the training run happens mostly in the training configuration file.
-Look at the examples of the full production configs for
-[Taskcluster](https://github.com/mozilla/translations/tree/main/configs/tc.prod.yml) and
-[Snakemake](https://github.com/mozilla/translations/tree/main/configs/config.prod.yml).
+The configuration of a training run happens mostly in a training configuration file.
+Look at the examples of the [full production configs for Taskcluster](https://github.com/mozilla/translations/tree/main/configs).
+
+See [Development documentation](development.md) for details on setting up the development environment.
 
 ## 1. Choose a language
 
 First, choose a language pair to train.
 
 Considerations:
 - The size of the parallel corpus on [OPUS](https://opus.nlpl.eu/)
-- Availability of monolingual data. The pipeline requires monolingual data in both source and target languages. 
-  Currently we support automatic donwloading only for [news crawl](https://data.statmt.org/news-crawl/)
-- Availability of [bicleaner-ai models](https://github.com/bitextor/bicleaner-ai-data/releases)
+- Availability of monolingual data. The pipeline requires monolingual data in both source and target languages.
+
+We use two-letter ISO 639-1 language codes, for example for Russian to English: `ru`, `en`.
+
+Explore available supported datasets:
+
+1. Go to [OPUS](https://opus.nlpl.eu/) and see how much parallel data is available for the language pair
+2. Use [find-corpus](https://github.com/mozilla/translations/tree/main/utils/find-corpus.py) tool. 
+Install [poetry](https://python-poetry.org/) and [Taskfile](https://taskfile.dev/installation/) first, then run:
+```bash
+task find-corpus -- ru en
+```
 
+It's hard to say how much parallel data is required to train something useful.
```

Comment:
> It's hard to say how much parallel data

This statement is ambiguous. I'd prefer to list our definitions of corpus sizes for small, medium, and large resources. I think this is a more helpful way to frame it. We can list issues as well for supporting low and high resource training. #803. Although I don't see one for high resource. We can also mention that corpus size doesn't guarantee a quality model, which I think what this statement is referring to.

Reply:
I replaced it to a table with clarification